app/controllers/staff/activities_controller.rb

```diff
@@ -4,7 +4,9 @@ class Staff::ActivitiesController < Staff::BaseController
   include Secured
 
   def index
```

Comment:
I'm wondering whether this is the right controller for this to go, as it's a list of only projects/third-party projects for the current user's organisation, but occupies a spot where I'd expect to find the code for viewing a list of *all* activities.

In the case of users, they can only ever belong to one organisation, so you could do have a top-level route:

`resources :projects, only: [:index, :show]`

And then have use the `current_user.organisation` in order to do the scoping?